jaseci_core/jaseci/__init__.py

```diff
@@ -9,4 +9,9 @@ def load_standard():
     import jaseci.actions.standard.jaseci  # noqa
 
 
+from .utils.jsorc import jsorc_start
+from .utils.utils import logger
+
+jsorc_start()
```

Comment:
@marsninja We decided to add the jsorc startup code in the __init__ of jaseci core here. We tried importing it in `mem_hook.py` first but only specific module of mem_hook are imported throughout jaseci (e.g. `from utils.mem_hook import BLAH) so it won't actually execute any "main" code in mem_hook.py. We added it here because it seems fitting. Let me know if there is a better place.